### DISCUSSION TITLE
Welcome to argocd-notifications Discussions!

### DISCUSSION THREAD

**Discussion by alexmt** [argoproj-labs/argocd-notifications#160] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jaipalkomakula** [comment]
Could you some one provide an example of slack incoming web-hook integration form Argocd notification ?
